src/main/java/com/threem/carrental/app/model/outbound/entity/MainOfficeEntity.java

```diff
@@ -29,11 +30,21 @@
     @Column(name = "address")
     private String address;
 
-    @OneToOne
-    @JoinColumn(name = "employee_id")
-    private EmployeeEntity employee;
+    @OneToOne(fetch = FetchType.LAZY)
```

Comment:
@miszalemko - everywhere we're using employee_id so please rename this field according to this template